pkg/test/pods.go

```diff
@@ -37,7 +37,15 @@ type PodOptions struct {
 	Conditions           []v1.PodCondition
 }
 
-func defaults(options PodOptions) *v1.Pod {
+// Pod creates a test pod with defaults that can be overriden by PodOptions.
+// Overrides are applied in order, with a last write wins semantic.
+func Pod(optionss ...PodOptions) *v1.Pod {
```

Comment:
nit: Is this normal? lol. Seems a little weird. Maybe `options` -> `opts` -> `o`?

Reply:
How about overrides.